### PR TITLE
Fc execution changes

### PR DESCRIPTION
**Pull request by gnivler**
Some minor adjustments to the FC functionality.  Started off by changing FC fallback after execution/abort to Off but then changed replaced that to using a field so it's user configurable.  Also made the lead time on attitude adjustment variable instead of hard coded for 3 minutes. The defaults are mimicking current functionality so nobody loses.  I didn't update the UI to control these settings.

### CODE REVIEW

**Comment by gnivler** [comment]
Forgot to mention that I changed the "maneuver complete" dV from 0.01m/s to 0.1/s, I don't think the extra granularity is doing any good and it might help reduce node chasing.

File: GameData/RemoteTech/Default_Settings.cfg [hunk 1]
```diff
@@ -18,6 +18,8 @@ RemoteTechSettings
 	HideGroundStationsOnDistance = True
 	ShowMouseOverInfoGroundStations = True
 	AutoInsertKaCAlerts = True
+	FCLeadTime = 180;
+    FCOffAfterExecute = false;
```
Comment (d4rksh4de) [inline GameData/RemoteTech/Default_Settings.cfg]: Spaces instead of TAB

File: src/RemoteTech/FlightComputer/Commands/ManeuverCommand.cs [hunk 2]
```diff
@@ -250,8 +258,8 @@ public override void Save(ConfigNode n, FlightComputer fc)
         /// <param name="computer">Current flightcomputer</param>
         public override void CommandEnqueued(FlightComputer computer)
         {
-            string KaCAddonLabel = String.Empty;
-            double timetoexec = (this.TimeStamp + this.ExtraDelay) - 180;
+			string KaCAddonLabel = String.Empty;
```
Comment (d4rksh4de) [inline src/RemoteTech/FlightComputer/Commands/ManeuverCommand.cs]: Indentation: TABs instead of spaces

**Comment by neitsa** [comment]
@gnivler : Thank you for your PR!

In the changed function, the comment says:

> When this command is aborted the fallback commnd is "Off".

However the field is set to false in your config file:

`FCOffAfterExecute = false;`

Which means the default is still `KillRot`and not `Off` (unless I'm mistaken). Could you change either the comment or the default value in the config file so they both match?

Otherwise I like this PR and will merge once this is fixed.

**Comment by gnivler** [comment]
Thanks @neitsa - I just adjusted the text to reflect that it defaults to KillRot, which matches the default config setting now.

**Comment by neitsa** [comment]
Perfect, thank you very much for your PR @gnivler! Merging now.

**Pull request merged**